addons/web_editor/static/tests/field_html_tests.js

```diff
@@ -132,6 +150,24 @@ QUnit.module('web_editor', {}, function () {
             form.destroy();
         });
 
+        QUnit.test('reading specially crafted value', async function (assert) {
```

Comment:
This test name could be improved to explain what it is testing. Something like "notebooks defined inside HTML field widgets are ignored when calling setLocalState".